pandas/tests/series/methods/test_convert_dtypes.py

```diff
@@ -246,3 +246,10 @@ def test_convert_dtypes(self, data, maindtype, params, answerdict):
 
         # Make sure original not changed
         tm.assert_series_equal(series, copy)
+
+    def test_convert_string_dtype(self):
+        # https://github.com/pandas-dev/pandas/issues/31731 -> converting columns
+        # that are already string dtype
+        df = pd.DataFrame({"A": ["a", "b", "c"], "B": ["ä", "ö", "ü"]}, dtype="string")
```

Comment:
can you parameterize using the nulls_fixture (and no null as you have now) as well

Reply:
we may want to have a null_fixture_for_strings e.g. [None, np.nan, pd.NA]

Reply:
There is no need to parametrize for nulls here, I think. I am creating this with dtype string, and once you have that dtype, there is no difference for different nulls. So it would be testing construction with different nulls, which isn't the goal here.

Reply:
are there other tests that convert_dtypes on strings with nulls?

Reply:
convert_dtypes doesn't care about nulls or not if the dtype is already "string".

But added one NA in the values.